digits/templates/models/gpu_utilization.html

```diff
@@ -21,3 +20,18 @@
     {% endif %}
 </dl>
 {% endfor %}
+{% if data_cpu %}
+<h3>CPU ({% if 'pid' in data_cpu %}#{{data_cpu.pid}}{% endif %})</h3>
```

Comment:
This is a little misleading. When I saw `CPU (#10291)` I thought it was talking about a CPU core or something. How about `Process 20291` instead?